packages/build-ui/src/pages/collection/page.rs

```diff
@@ -0,0 +1,320 @@
+                    div { class: "relative flex-1 mr-4",
+                        div { class: "absolute inset-y-0 left-0 flex items-center pl-3 pointer-events-none",
+                        edit::Search {    class: "text-white",}
+                        }
+                        input {
+                            class: "bg-[#222] border border-[#333] text-white text-sm rounded-none block w-full pl-10 p-2.5",
+                            placeholder: "Search by title",
+                            r#type: "text"
+                        }
+                    }
+                    
+                    button { class: "bg-[#222] border border-[#333] text-white px-4 py-2 flex items-center justify-center w-full sm:w-auto",
+                    onclick: move |_| modal_state.set(ModalState::NewCollection),
+                    folder::UploadFolder { 
+                        class:"mr-3"
+                     },
+                        "New Collection"
+                    }
+                }
+                
+                // Content area (FilterSidebar and Table)
+                div { class: "flex flex-col md:flex-row flex-1 w-full",
+                    // FilterSidebar (hidden on small screens unless toggled)
+                    if *show_filters.read() {
+                        div { 
+                            class: format!(
+                                "w-64 bg-[#171717] border-r border-[#333] fixed inset-y-0 left-0 z-40 transform {} md:relative md:z-auto md:translate-x-0 transition-transform duration-300",
+                                if *show_filters.read() { "translate-x-0" } else { "-translate-x-full" }
+                            ),
+                            FilterSidebar {}
+                        }
+                    }
+                    
+                    // Table body
+                    div { class: "flex-1 overflow-auto",
+                        table { class: "w-full text-sm text-left border-collapse min-w-[800px]",
+                            // Table header and body content...
+                            // (Keeping the existing table code)
+                            
+                            // Table header
+                            thead { class: "text-xs uppercase bg-[#1a1a1a] border-b border-[#333]",
+                                tr {
+                                    th { class: "px-2 py-2 sm:px-4 sm:py-3 whitespace-nowrapp", 
+                                        div { class: "flex items-center", 
+                                            span { "#" }
+                                            span { class: "ml-1 text-gray-500", "%" }
+                                        }
+                                    }
+                                    th { class: "px-2 py-2 sm:px-4 sm:py-3 whitespace-nowrap", 
+                                        div { class: "flex items-center", 
+                                            span { "Collection" }
```

Comment:
```suggestion
                                            span { "Collection" }
```
please use `dioxus_translate`
- Refer to https://github.com/biyard/ratel/blob/b1ba98d4f1f16ad29993d5edfbfc61bbc9f7fa03/packages/main-ui/src/pages/politicians/i18n.rs#L3
- Basic composition of a page: https://github.com/biyard/ratel/blob/dev/packages/main-0ui/src/pages/politicians/mod.rs#L3-L5
   - i18n: translation
   - controller: Controller
   - page: a page
   - components: a directory of components
- Also you can see https://docs.biyard.co/web/package-structure.html